### PR TITLE
Feature/last sass

### PR DESCRIPTION
**Pull request by battan40**
# Description of Changes

- This branch adds our mixins file 
- Imports files properly

### CODE REVIEW

File: src/css/_mixins.scss [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-//   }
-//
-//   @if $margin != 0 {
-//     margin: $margin;
-//   }
-// }
-//There are 12 display: flex; flex-direciton: row; rules in the index.scss file
-//The proportions for margin need to stay simple, so it can't go in all 12... but this could be the instance of refactor to add a uniform mixin
+@mixin
+centerFlexWithPaddingAndMargin($padding: 0, $margin: 0, $direction: center) {
```
Comment (rhen92) [inline src/css/_mixins.scss:2]: I like how you condensed this mixin from the original!

**Comment by rhen92** [comment]
I like how you condensed the scss file with the mixins!! Great job!

File: src/css/_mixins.scss [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-//this mixin is not currently applied
+@mixin
+displayFlexRow($direction: row) {
+  display: flex;
+  flex-direction: $direction;
+}
 
-//possible discussion point here:  should we make this an extend instead?
+@mixin
```
Comment (mistercanderson) [inline src/css/_mixins.scss:15]: I like the use of mixins here, nice work

File: src/css/index.scss [hunk 3]
```diff
@@ -265,11 +261,10 @@ li {
 
 /* ~~~~~~~~ RECIPE CARDS ~~~~~~~~~ */
 main {
+  @include centerFlexWithPaddingAndMargin();
+  @include displayFlexRow();
```
Comment (mistercanderson) [inline src/css/index.scss:265]: putting those mixins to good use here

**Review by mistercanderson — APPROVED** [review]
This CSS looks syntactically awesome & SASSY

**Pull request merged**